Ambiguous-Squids/bangazon-api
# Rw products serializer

## Description

created the ProductType, PaymentType, and Order serializers with the hyperlink serialization method django provides.
The description that you provide should be comprehensive enough to...


## Related Issue
 Please link to the issue number here: 
#31 

#33 

#36 

## Motivation and Context
 Why is this change required? What problem does it solve? 

## How Has This Been Tested?
tested in python shell

## Steps to Test
run python shell and import necessary modules

System configuration.
3rd party libraries that need to be installed.
Command line utilities to run.
If there is a UI component, give clear instructions for steps to perform in the UI, and what they should expect to see as the outcome of those steps.

## Types of changes
 What types of changes does your code introduce? Put an \`x\` in all the boxes that apply: 
- [ x] New feature (non-breaking change which adds functionality)

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help! 
- [x ] My code follows the code style of this project.
- [ x] I have updated the documentation accordingly.
